_includes/footer.html

```diff
@@ -0,0 +1,18 @@
+<footer class="simple_footer">
+  <img class="leaf_right" src="../../img/home_one/leaf_footter.png" alt="">
+  <div class="container ">
+    <div class="row">
+      <div class="col-lg-6">
+        <p>© 2021 All Rights Reserved by <a href="#">Lovelace Academy</a></p>
+      </div>
+      <div class="col-lg-6 text-right">
+        <ul class="list-unstyled f_social_icon">
+          <li><a href="#"><i class="social_youtube"></i></a></li>
```

Comment:
I had working URLs in a newer commit - can you please update? 
https://github.com/LovelaceAcademy/Learn/blob/310c596f0186ad784d966a9859e7b7d7c47822f5/index.html#L706 
https://github.com/LovelaceAcademy/Learn/blob/310c596f0186ad784d966a9859e7b7d7c47822f5/index.html#L707
https://github.com/LovelaceAcademy/Learn/blob/310c596f0186ad784d966a9859e7b7d7c47822f5/index.html#L708